src/chunk.rs

```diff
@@ -202,3 +204,65 @@ pub(crate) fn update_chunk_mesh(
     }
 
 }
+
+pub(crate) fn update_chunk_visibility(
+    camera: Query<(&Camera, &OrthographicProjection, &Transform)>,
+    mut chunks: Query<(&Chunk, &mut Visible)>,
+) {
+    if let Some((_current_camera, ortho, camera_transform)) = camera.iter().find(|data|
+        if let Some(name) = &data.0.name {
+            name == CAMERA_2D
+        } else { false }
+    ) {
+        // Transform camera into world space.
+        let left = camera_transform.translation.x + (ortho.left * camera_transform.scale.x);
+        let right = camera_transform.translation.x + (ortho.right * camera_transform.scale.x);
+        let bottom = camera_transform.translation.y + (ortho.bottom * camera_transform.scale.x);
+        let top = camera_transform.translation.y + (ortho.top * camera_transform.scale.x);
```

Comment:
I think this is an error, you have `camera_transform.scale.x` even for `top` and `bottom`, and I mean I don't think the scale would be different on the x and y but for being safe this should probably be changed to `camera_transform.scale.y` in the top and bottom parts